AMEBI.Domain/Services/UserService.cs

```diff
@@ -0,0 +1,80 @@
+using System.Threading.Tasks;
+using AMEBI.Domain.EF;
+using AMEBI.Domain.Model;
+using Microsoft.AspNetCore.Identity;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using System;
+using Microsoft.Extensions.Configuration;
+using System.IO;
+using Novell.Directory.Ldap;
+using AMEBI.Domain.LDAP;
+using Microsoft.Extensions.Options;
+
+namespace AMEBI.Domain.Services
+{
+    public class UserService : IUserService
+    {
+        private readonly Context _context;
+        private readonly LdapConfig _config;
+
+        public UserService(Context context, IOptions<LdapConfig> config)
+        {
+            _context = context;
+            _config = config.Value;
+        }
+
+        public async Task<User> FindAsync(string username)
+            => await _context.Users.SingleOrDefaultAsync(x => x.Username == username); 
+        
+        public async Task AddAsync(string username, string password)
+        {
+            var user = new User(username, password);
+
+            await _context.Users.AddAsync(user);
+            await _context.SaveChangesAsync();
+        }
+
+        public void LoginAsync(string username, string password)
+        {
+            using (var connection = new LdapConnection())
+            {
+                connection.ConnectionTimeout = 10000;
```

Comment:
Consider moving it to class constants or maybe even make it configurable from appsettings